homework1/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Runtime.InteropServices;
+
+namespace homework1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
What has you tried to do here? it seems that not in accordance with a task.....

Reply:
I'm sorry I noticed there might have been a misunderstanding. Could you please clarify what you meant?